data-raw/01_gut_compare_df.R

```diff
@@ -0,0 +1,2 @@
+gut_compare_df <- sourmashconsumr::read_compare_csv("inst/extdata/gut_compare.csv.gz")
```

Comment:
How are these files created and/or downloaded? I don't think these are from the sourmash commands file right?

And what does `inst` stand for?

Reply:
> How are these files created and/or downloaded? I don't think these are from the sourmash commands file right?

Ya this PR touches a lot of files so it's kinda hard to follow, but I'll do my best at explaining it!
1. `inst/extdata` contains any non-R formatted data that might be useful for demonstrating how your package works. In this case, because the package reads in text files, I need those files to live in `inst/extdata`
1. The `data-raw` folder is used to document how R formatted data that is accessible to the package is generated (typically called by the user with the `data` function, so `data(gut_compare_df)`. In this case, the only thing I did to create those objects in R is read in a CSV vile to an R object, then write the R object out to an Rda file.
2. Given this, I wanted to also document where the csv files came from. I used the scripts `data-raw/00_sourmash_commands.sh` to do that. It documents the version of sourmash I used to generate the files and then documents the shell commands one would need to run to re-generate all of the files in `inst/extdata`.

> And what does `inst` stand for?

I don't actually know what `inst` stands for, but if data is in `inst/extadata` when the package is built, the external data is then available at `extdata` to the user. Following conventions laid out here: https://r-pkgs.org/data.html